src/ui/public/modals/index.js

```diff
@@ -0,0 +1,3 @@
+import './confirm_modal';
+import './safe_confirm';
```

Comment:
It was always a modal, it's just that now it uses our custom modal.  Even if we left it using the native modal I think it could still go in a modals folder.

There is nothing else that went in the previous safe_confirm folder (except tests) so it doesn't deserve a dedicated extra folder IMO.  Is there another location you think it could go that isn't in it's own dedicated folder?